lib/miq_cockpit.rb

```diff
@@ -16,6 +16,7 @@ class WS
 
     def self.can_start_cockpit_ws?
       MiqEnvironment::Command.is_linux? &&
+        !MiqEnvironment::Command.is_podified? &&
         File.exist?(COCKPIT_WS_PATH) &&
         File.exist?(COCKPIT_SSH_PATH)
```

Comment:
This is used by https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_cockpit_ws_worker.rb#L16-L18